spec/request/lifecycle/service_route_bindings_synoptic_spec.rb

```diff
@@ -0,0 +1,284 @@
+require 'spec_helper'
+require 'cloud_controller'
+
+# This lifecycle test aims to use different v3 service endpoints together
+RSpec.describe 'V3 service route bindings synoptic' do
+  before do
+    stub_request(:get, 'http://example.org/amazing-service-broker/v2/catalog').
+      with(basic_auth: %w(admin password)).
+      to_return(status: 200, body: catalog, headers: {})
+
+    stub_request(:put, %r{\Ahttp://example.org/amazing-service-broker/v2/service_instances/.+\z}).
+      with(basic_auth: %w(admin password)).
+      to_return(status: 201, body: {}.to_json, headers: {})
+
+    stub_request(:get, 'https://master.default.svc.cluster-domain.example/apis/networking.cloudfoundry.org/v1alpha1').
+      with(basic_auth: %w(admin password)).
+      to_return(status: 200, body: '', headers: {})
+
+    stub_request(:get, %r{\Ahttp://example.org/amazing-service-broker/v2/service_instances/.+/service_bindings/.+\z}).
+      with(basic_auth: %w(admin password)).
+      to_return(status: 200, body: { parameters: { key1: 'value1', key2: 'value2' } }.to_json, headers: {})
+
+    stub_request(:delete, %r{\Ahttp://example.org/amazing-service-broker/v2/service_instances/.+/service_bindings/.+\z}).
+      with(basic_auth: %w(admin password)).
+      to_return(status: 410, body: '', headers: {})
+    VCAP::CloudController::Config.config.set(:kubernetes, nil)
+  end
+
+  it 'works end to end' do
+    org_guid = create_org
+    space_guid = create_space(org_guid)
+
+    post '/v3/service_brokers', create_service_broker_request_body.to_json, admin_headers
+    expect(last_response).to have_status_code(202)
+    execute_all_jobs(expected_successes: 1, expected_failures: 0)
+
+    plan_guid = get_route_service_plan using: admin_headers
```

Comment:
I like how having a named parameter and dropping the parentheses makes this very readable.